sycl/plugins/unified_runtime/ur/adapters/level_zero/device.cpp

```diff
@@ -340,6 +340,14 @@ UR_APIEXPORT ur_result_t UR_APICALL urDeviceGetInfo(
       return UR_RESULT_SUCCESS;
     }
 
+    if (Device->isCCS()) {
+      ur_device_partition_property_t cslice = {
```

Comment:
Did it not work without the if condition? It's not a big deal, but I guess the code would be a bit cleaner without hardcoding the struct.

For context, I'm referring to this comment: https://github.com/jandres742/llvm/commit/5f45a51c78e571551b91806942b30cdbaf569370#r122951465

Reply:
sorry @fabiomestre, had missed that comment.

Confirmed this is still needed for:

> SYCL_PI_LEVEL_ZERO_EXPOSE_CSLICE_IN_AFFINITY_PARTITIONING=1 ZEX_NUMBER_OF_CCS=0:4 ONEAPI_DEVICE_SELECTOR=ext_oneapi_level_zero:gpu ./level_zero_ext_intel_cslice.run

Reply:
Fair enough :+1: